10.9-libcxx/stable/main/finkinfo/languages/pip-py.info

```diff
@@ -1,17 +1,19 @@
-Info2: << 
+# -*- coding: ascii; tab-width: 4 -*-
```

Comment:
Should these emacs settings really be in the files?

Reply:
I've found them in most of the newer `.info` files, so assumed it was policy now. Don't really care, since I am using vi for editing them most of the time (and personally prefer 2-4 spaces indents...), so I am flexible in that respect.

Reply:
I add them to my packages. BBEdit uses them. I like forcing ascii since info files should always be ascii and I like 4 space indents. :) Feel free to use it or not in your own packages. It's not policy.